From 83c0aef587c7d86819376b35003dc5f6c62ab72d Mon Sep 17 00:00:00 2001
From: Sebastian Kemper <sebastian_ml@gmx.net>
Date: Tue, 8 Nov 2022 21:10:52 +0100
Subject: [PATCH 1/2] baresip: use return in init script

"start_service()" is a function, hence "return" should be used instead
of "exit".

Signed-off-by: Sebastian Kemper <sebastian_ml@gmx.net>
(cherry picked from commit b69ae7b9344228e1e83aaee2f62cebbd599c1388)
Signed-off-by: Richard Muzik <richard.muzik@nic.cz>
---
 net/baresip/files/baresip.init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/baresip/files/baresip.init b/net/baresip/files/baresip.init
index 72149be..d48e5d6 100644
--- a/net/baresip/files/baresip.init
+++ b/net/baresip/files/baresip.init
@@ -21,7 +21,7 @@ start_service() {
   if [ "$ENABLE_BARESIP" != yes ]; then
     $LOGGER User configuration incomplete - not starting $DAEMON
     $LOGGER Check ENABLE_BARESIP in $DEFAULT
-    exit 1
+    return 1
   fi
 
   procd_open_instance
-- 
2.45.2

